Documentation/gettingstarted/kind.rst

```diff
@@ -23,17 +22,9 @@ Install Cilium
 ==============
 
 .. include:: k8s-install-download-release.rst
+.. include:: kind-preload.rst
```

Comment:
Is this file intended to either include or point to the `kind-install.rst`? I missed where that takes place.

Reply:
This is just the instructions to pre-load the cilium image into the kind cluster. As this is required for both the Hubble and Kind getting started guide, I've split this into a separate file that is included in both gsgs.

Reply:
Not sure how I missed it before, now I look it's definitely there. :+1: